src/runtime/wasm-tools.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * @license
+ * Copyright (c) 2019 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+import {Schema} from './schema.js';
+import { toProtoJSON } from './wasm.js';
+import proto_target from 'protobufjs/cli/targets/proto.js';
+import Root from 'protobufjs/src/root.js';
+
+// Convert a Schema to .proto format that can be used to compile protobuf wrappers
+export async function toProtoFile(schema: Schema) {
```

Comment:
It's a text proto description. Yes, I could assert that the text output equals a handcoded fixture, but it would mean updating the fixture everytime the test inputs are updated and wouldn't test if the proto def actually round trips. 

What looking at working on next is a C++ wasm manual integration test that if node is launched with --expose wasm, we can test the round trip from protobufjs to C++ protobuf compiler.  I'm happy to nix the roundtrip on this test if you want in favor of a C++ end-to-end test.

Reply:
I think we want a fairly well-defined translation from schema fields to proto fields, and a unit test that lays that out explicitly would be useful. Having to update the test to match when the logic changes is a feature, IMO.

Testing that the proto round-trips is a separate concern (and we can have that, but as a separate test).